ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/task/GetBlockFromPeerTask.java

```diff
@@ -111,6 +116,7 @@ protected void executeTask() {
   private CompletableFuture<PeerTaskResult<List<Block>>> completeBlock(
       final PeerTaskResult<List<BlockHeader>> headerResult) {
     if (headerResult.getResult().isEmpty()) {
+      LOG.debug("GetBlockFromPeerTask: header result is empty.");
```

Comment:
I think we missed the name of this class

Reply:
ups! Done!